templates/zerver/emails/confirm_registration_mit.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
Found a cleaner reference point here: https://github.com/zulip/zulip/blob/44113d6436da02ef4126aab8342befe53605fe8d/templates/zerver/base.html. For instance, the next line SHOULD be `<html lang='{{LANGUAGE_CODE}}'>`.